apps/web/lib/web/router.ex

```diff
@@ -44,6 +44,7 @@ defmodule Web.Router do
       # Gateways
       live "/gateways", GatewaysLive.Index
       live "/gateways/new", GatewaysLive.New
+      live "/gateways/:id/edit", GatewaysLive.Edit
```

Comment:
For changing a deployed Gateway's Name and other non-connectivity-related actions.